tests/src/Unit/Service/Helper/SettingsHelperTest.php

```diff
@@ -235,4 +236,82 @@ public function providerTestIsValidContentReplacementMode() {
 
     return $data;
   }
+
+  /**
+   * Tests the getUdfLimitsForType() method.
+   *
+   * @covers ::getUdfLimitsForType
+   *
+   * @param string $test_value
+   * @param boolean $expected
+   *
+   * @dataProvider providerTestGetUdfLimitsForType
+   */
+  public function testGetUdfLimitsForType($test_value, $expected) {
```

Comment:
getUdfLimitsForType() tests are not related or required, but easy to get to and good for coverage, so I did it quickly on the side.